docs/user_guide.rst

```diff
@@ -133,14 +133,24 @@ circuit(s) are sampled:
    `Inspecting circuits before submitting them for execution`_ for inspecting the actual run request sent for
    execution.
 
+The calibration data for an IQM quantum computer is stored in a calibration set. An :class:`.IQMBackend` instance
+always corresponds to a specific calibration set, so that its transpilation target uses only those QPU components
+(qubits and computational resonators) and gates which are available in that calibration set. The server default
+calibration set will be used by default, but you can also use a different calibration set by specifying the
+``calibration_set_id`` parameter for :meth:`.IQMProvider.get_backend` or :class:`.IQMBackend`. If the server default
+calibration set has changed after you created the backend, the backend would still use the original default calibration
+set when submitting circuits. In this case, a warning will be shown to notify you about this, and you would need to
+create a new backend if you want to use the newer calibration set.
```

Comment:
```suggestion
calibration set has changed after you have created the backend, the backend will still use the original default calibration
set when submitting circuits for execution. In this case, a warning will notify you about this.
You will need to create a new backend if you want to use the new default calibration set.
```

Reply:
Sorry, merged already, can be done in https://github.com/iqm-finland/qiskit-on-iqm/pull/123 I guess.

Reply:
Applied